### PR TITLE
Added ThresholdBar chart

### PR DESCRIPTION
**Pull request by shumber**
ThresholdBar chart added. Colours bars based on threshold value.

### CODE REVIEW

File: Swift Charts Examples.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -556,6 +560,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Swift Charts Examples/Preview Content\"";
+				DEVELOPMENT_TEAM = LUMDJJBX94;
```
Comment (atrinh0) [inline Swift Charts Examples.xcodeproj/project.pbxproj:563]: Should avoid committing these changes

File: Swift Charts Examples/Charts/BarCharts/ThresholdBar.swift [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        if lastSales >= dataFilterThreshold && salesData.sales < dataFilterThreshold { //Transition from high to low
+            let dataSet = segmentedChartData(day: salesData.day, sales: salesData.sales, aboveThreshold: false, fillColor: .blue)
+            filteredSegments.append(dataSegment)
+            dataSegment.removeAll()
+            dataSegment.append(dataSet)
+            lastSales = salesData.sales
+        }
+    }
+    return filteredSegments
+}
```
Comment (atrinh0) [inline Swift Charts Examples/Charts/BarCharts/ThresholdBar.swift:126]: These should live in an enum namespace to remove them from being globally accessed

**Comment by atrinh0** [comment]
> ThresholdBar chart added. Colours bars based on threshold value.

This can be achieved more simply via, https://github.com/jordibruin/Swift-Charts-Examples/pull/50

File: Swift Charts Examples.xcodeproj/project.pbxproj (hunk 1 above)
Comment (shumber) [reply Swift Charts Examples.xcodeproj/project.pbxproj:563]: Thanks for the info! Quite new to this.

**Pull request closed without merging**